### PR TITLE
Changes from Dev branch.

### PR DESCRIPTION
**Pull request by xarrijorge**
Compete site changes made on from the dev branch. The project has reached a stage we think it should be merged into master.

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,262 @@
+body{
+  /* box-sizing: border-box; */
```
Comment (zlayabekrija) [inline index.css]: please remove commented out properties.

File: index.css [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+  font-size: 1.3em;
+  font-weight: 500;
+  margin: 0;
+}
+
+.right-action-buttons{
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  /* border: 2px solid black; */
```
Comment (zlayabekrija) [inline index.css]: same here

File: index.css [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+  height: 100vh;
+}
+aside{
+  width: 25%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: start;
+  margin: 0 2%;
+  /* padding-right: 5%; */
```
Comment (zlayabekrija) [inline index.css]: and here

File: index.css [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+}
+
+.thumbnails {
+  display:flex;
+  /* border:solid 1px; */
```
Comment (zlayabekrija) [inline index.css]: and here

File: index.css [hunk 5]
```diff
@@ -0,0 +1,262 @@
...
+}
+.thumbnails img{
+  min-width: 100%;
+}
+
+.inner-left img {
+
+}
+.inner-left{
+  /* width: 169px;
```
Comment (zlayabekrija) [inline index.css]: here as well

File: index.css [hunk 6]
```diff
@@ -0,0 +1,262 @@
...
+  height: 95px; */
+  /* border: solid 1px; */
+  margin-right:5px;
+}
+.inner-right{
+  width: 100%;
+  /*  height: 90px; */
```
Comment (zlayabekrija) [inline index.css]: here :)

File: index.css [hunk 7]
```diff
@@ -0,0 +1,262 @@
...
+  clear: both;
+}
+
+.autoplay { 
+color: rgba(39, 38, 38, 0.877); 
+font-size:1vw; margin-bottom: 5px; }
+
+.autoplay ul {list-style: none; display: flex; flex-direction: row; justify-content: space-between;}
+
+.comment-stats-section{
```
Comment (zlayabekrija) [inline index.css]: class is empty please remove it.

File: index.css [hunk 8]
```diff
@@ -0,0 +1,262 @@
...
+  
+  /* border: solid 1px; */
+
+}
+
+.video-stats{
+   /* min-height: 40vh;  */
```
Comment (zlayabekrija) [inline index.css]: please remove these properties

File: index.css [hunk 9]
```diff
@@ -0,0 +1,262 @@
...
+  line-height: 0;
+  letter-spacing: -0.4em;
+  color: #909090;
+}
+.left-reviews{
+  text-align: left;
+}
+
+
+.video-comments{
```
Comment (zlayabekrija) [inline index.css]: again empty class, please remove it.

File: index.css [hunk 10]
```diff
@@ -0,0 +1,262 @@
...
+  margin-bottom: 30px;
+}
+
+ .author_avatar{
+  width: 8%;
+}
+.author_avatar img{
+  width: 100%;
+  margin-top: -10px;
+  /* height: 80%; */
```
Comment (zlayabekrija) [inline index.css]: remove this please.

**Comment by xarrijorge** [comment]
Closing this to open a new PR with the edits added.

**Pull request closed without merging**